projects/fsastorefrontlib/src/cms-components/my-account/policy/policies-chart/policies-chart.component.html

```diff
@@ -10,7 +10,7 @@
           'fscommon.paymentFrequency' | cxTranslate
         }}</span>
         <ng-select
-          [(ngModel)]="selectedFrequency.frequency"
+          [placeholder]="selectedFrequency.frequency"
```

Comment:
Use `placeholder` to initially select first item from dropdown. `ngModel` from some reason affects `selectedItem` emitted by `ng-select` component.